package.json

```diff
@@ -444,6 +444,30 @@
             {
                 "type": "python"
             }
+        ],
+        "languageModelTools": [
+            {
+                "name": "python_get_packages",
+                "displayName": "Get Python Packages",
+                "modelDescription": "Given a file path, finds the workspace and selected python environment for this file, and returns the list of installed packages.",
```

Comment:
"finds the workspace and selected python environment for this file" may be unnecessary, and you may want to tell the model when this should be called. Something like, "Returns the packages available to the given Python file. You should call this when you want to generate Python code in order to determine the packages the user prefers to use."